src/script/components/panel/UserActions.tsx

```diff
@@ -367,7 +368,9 @@ const UserActions: React.FC<UserActionsProps> = ({
       onCancel={onAction}
     />
   ) : (
-    <PanelActions items={items} />
+    <ul css={listCSS}>
+      <PanelActions items={items} />
+    </ul>
```

Comment:
Wouldn't it be simpler to leave `<ul>` inside PanelActions? I found it a little bit tricky that the PanelActions component returns a list of `<li>`, and has to be wrapped with an additional `<ul>` when called (because `<li>` can't be used outside of `<ul>`). And also it uses styles from the `PanelActions.styles`. What's the difference there?

Reply:
By 2nd place you mean `ConversationDetails.tsx`? I believe the parent there is `FadingScrollbar` which has a `<div>` iside

Reply:
Ohh I see now :) Never mind then!